src/commands/new.ts

```diff
@@ -0,0 +1,182 @@
+import isIgniteDirectory from '../lib/is-ignite-directory'
```

Comment:
This command had the most significant changes, I think.

Reply:
Diff: https://github.com/infinitered/ignite/pull/1379/commits/7f86d1f0e9f80d91782d26d58aa34f46bd5b8a16#diff-6900e9ede7ad7ad9c9e77f95473dca32